garak/resources/cohere/scripts/generate_sample.py

```diff
@@ -0,0 +1,23 @@
+import os
```

Comment:
Looks like example code perhaps? Can a brief comment be added to document what this file is for?